.github/workflows/tests.yml

```diff
@@ -30,7 +30,7 @@ env:
   PYTEST_ADDOPTS: "--color=yes"
   PYTHON_VERSION: "3.10"
   TF_PLUGIN_CACHE_DIR: "/home/runner/.terraform.d/plugin-cache"
-  TF_VERSION: 1.3.0
```

Comment:
Can you update the version in linting.yml too?

Reply:
I will send a separate PR updating versions and CI